### PR TITLE
Final fixes 53

### PR DESCRIPTION
**Pull request by madhavmansuriya40**

### CODE REVIEW

File: src/app/fyle/my-expenses/my-expenses.page.ts [hunk 1]
```diff
@@ -625,7 +625,7 @@ export class MyExpensesPage implements OnInit {
     });
 
     if (popupResults === 'primary') {
-      from(this.loaderService.showLoader()).pipe(
+      from(this.loaderService.showLoader('', 2500)).pipe(
```
Comment (Julias0) [inline src/app/fyle/my-expenses/my-expenses.page.ts:628]: dont pass empty string - 
pass `Deleting Expense`

File: src/app/fyle/view-team-trip/advance-request/advance-request.component.html [hunk 2]
```diff
@@ -43,29 +45,6 @@
           </div>
         </div>
 
-        <div class="advance-requests--element" *ngFor="let customField of advanceRequest.custom_field_values">
```
Comment (Julias0) [inline src/app/fyle/view-team-trip/advance-request/advance-request.component.html:46]: why remove the custom fields?

Comment (madhavmansuriya40) [reply src/app/fyle/view-team-trip/advance-request/advance-request.component.html:46]: it was showing custom fields 2 times, can check the code block just above this

this was done by mistake (copy paste
)

Comment (Julias0) [reply src/app/fyle/view-team-trip/advance-request/advance-request.component.html:46]: -_-

**Review by Julias0 — APPROVED** [review]

**Pull request merged**